Resources/Prototypes/SS220/Entities/Clothing/Uniforms/nt_NO_uniform.yml

```
@@ -0,0 +1,12 @@
+# © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+- type: entity
+  parent: UnsensoredClothingUniformBase
+  id: ClothingUniformNanoTrasenNavyOfficer
+  name: униформа офицера NanoTrasen
+  description: На этой униформе изображено звание «лейтенант-коммандер». Носят исключительно офицеры Нанотрейзен. Для защиты используются экзотические материалы.
```

```suggestion
  description: На этой униформе изображено звание «лейтенант-коммандер». Её носят исключительно офицеры NanoTrasen. Защитные свойства форме придают экзотические материалы.
```
Для защиты кого? формы?